react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-
 import styles from './styles.module.scss';
 
-const Square = () => (
-  <button type="button" className={styles.square}>
-    {/* TODO */}
-  </button>
-);
+function Square({ onClick,value }) {
```

Comment:
And here create a class where you can handle the onClick which call the onClick that you receive like a prop and  with the 'i' that you receive like a prop too.